addons/website_crm_partner_assign/models/res_partner_grade.py

```diff
@@ -6,13 +6,8 @@
 
 class ResPartnerGrade(models.Model):
     _name = 'res.partner.grade'
-    _order = 'sequence'
-    _inherit = ['website.published.mixin']
-    _description = 'Partner Grade'
+    _inherit = ['res.partner.grade', 'website.published.mixin']
 
-    sequence = fields.Integer('Sequence')
```

Comment:
Could you make a clean [MOV] commit independently from the rest, so that we better understand what is moved, and what is changed / new / removed ? Thanks !

Reply:
Done